AudioSystem/Editor/AudioConfigSequenceEditor.cs

```diff
@@ -45,7 +45,7 @@ private void DrawEntry(SerializedProperty property)
 				}
 				EditorGUILayout.EndHorizontal();
 
-				var postClipDelay = property.FindPropertyRelative("postClipDelay");
+				var postClipDelay = property.FindPropertyRelative("preClipDelay");
```

Comment:
Need matching the name. `var preClipDelay`

Reply:
Done